scripts/dist.ts

```diff
@@ -88,22 +88,34 @@ function toDist(sourcePath: string): YAML.Document {
     status: yaml.toJS().status,
   };
 
+  const allCompatFeatures: string[] = Array.isArray(overridden.compatFeatures)
```

Comment:
I'd like to rebase this on top of https://github.com/web-platform-dx/web-features/pull/1233 because I think it'll look nicer after that change.